tests/ImageSharp.Tests/TestImages.cs

```diff
@@ -25,6 +25,9 @@ public static class Png
             public const string Powerpoint = "Png/pp.png";
             public const string SplashInterlaced = "Png/splash-interlaced.png";
             public const string Interlaced = "Png/interlaced.png";
+            public const string Palette8Bpp = "Png/Palette-8bpp.png";
```

Comment:
Unix systems are case-sensitive, so you have to rename this to `palette-8pp.png`, to match the filename. There is a [test failure](https://travis-ci.org/SixLabors/ImageSharp/builds/267901746#L2360) on travis because of that.